packages/renderer/src/lib/pod/PodDetailsKube.svelte

```diff
@@ -25,5 +26,9 @@ onMount(async () => {
 </script>
 
 {#if kubeDetails}
-  <MonacoEditor content="{kubeDetails}" language="yaml" />
+  {#if pod.kind === 'podman'}
+    <MonacoEditor content="{kubeDetails}" language="yaml" />
+  {:else}
+    <KubeEditYAML content="{kubeDetails}" />
```

Comment:
Do you think it would be helpful for the user to have a short paragraph here explaining what will be the operation run into the cluster when pressing "Edit"?

Some explanation saying it will be similar to `kubectl apply -f` for example